Resources/Locale/en-US/devices/network-configurator.ftl

```diff
@@ -24,3 +33,13 @@ network-configurator-tooltip-edit = Edit targets device list
 network-configurator-tooltip-clear = Clear targets device list
 network-configurator-tooltip-copy = Copy targets device list to multitool
 network-configurator-tooltip-show = Show a holographic visualization of targets device list
+
+# examine
+network-configurator-examine-mode-link = [color=red]Link[/color]
+network-configurator-examine-mode-list = [color=green]List[/color]
+network-configurator-examine-current-mode = Current mode: {$mode}
+network-configurator-examine-switch-modes = Press {$key} to switch modes
+
+# item status
+network-configurator-item-status-label = Current mode: {$mode}
+{$keybinding} to switch mode
```

Comment:
oop

Reply:
That's actually valid ftl
The linebreak carries over into the ui where that line is displayed